pkg/client/cmd/deploy.go

```diff
@@ -7,21 +7,54 @@ import (
 	"io"
 	"os"
 	"path/filepath"
+	"sort"
+	"strconv"
 	"strings"
+	"time"
 
 	"github.com/fatih/color"
 	"github.com/luizalabs/teresa/pkg/client"
 	"github.com/luizalabs/teresa/pkg/client/connection"
 	"github.com/luizalabs/teresa/pkg/client/tar"
 	dpb "github.com/luizalabs/teresa/pkg/protobuf/deploy"
+	"github.com/olekukonko/tablewriter"
 	"github.com/satori/go.uuid"
 	"github.com/spf13/cobra"
 	context "golang.org/x/net/context"
 	"golang.org/x/sync/errgroup"
 )
 
+type ByRevision []*dpb.ListResponse_Deploy
```

Comment:
What do you think about moving this type to `pkg/client/deploy` or something like this? I'm trying to avoid much logic on `cmd` pkg.